### PR TITLE
Nyt county

### PR DESCRIPTION
**Pull request by sglyon**

### CODE REVIEW

**Comment by sglyon** [comment]
oops my bad for bunding in the new fips column -- you can review it here I guess

**Comment by sglyon** [comment]
I had to push some updates to the db/schemas directory.

We got some commands out of order when we added new tables recently so I had to update the order

File: db/schemas/026_covid_us_mat_view.sql [hunk 1]
```diff
@@ -12,7 +12,11 @@ CREATE OR REPLACE VIEW api.covid_us AS
...
     cv.name AS variable,
-    uc.value
+    uc.value,
+    CASE WHEN fips < 100 THEN LPAD(fips::TEXT, 2, '0') ELSE LPAD(fips::TEXT, 5, '0') END AS fips
    FROM data.us_covid_last_vintage lv
      LEFT JOIN data.us_covid uc USING (dt, fips, variable_id, vintage)
      LEFT JOIN meta.covid_variables cv ON cv.id = uc.variable_id;
+
+COMMENT ON COLUMN api.covid_us.location IS E'An integer identifying the geography. For US states and counties this is a fips code without leading zeros.';
+COMMENT ON COLUMN api.covid_us.location IS E'The 2 digit fips code for US states and 5 digit fips code for US counties.';
```
Comment (cc7768) [inline db/schemas/026_covid_us_mat_view.sql]: Should this be `fips` rather than `location`?

File: src/cmdc_tools/datasets/nytimes/data.py [hunk 2]
```diff
@@ -29,7 +30,7 @@ def _insert_query(self, df: pd.DataFrame, table_name: str, temp_name: str, pk: s
 
     def get(self):
         df = pd.read_csv(self.url, parse_dates=["date"])
-        df["vintage"] = pd.datetime.today().date()
+        df["vintage"] = self._retrieve_vintage()
```
Comment (cc7768) [inline src/cmdc_tools/datasets/nytimes/data.py:33]: I changed this to use the `_retrieve_vintage` method and moved the vintage methods to the `DatasetBase` class

File: db/schemas/026_covid_us_mat_view.sql (hunk 1 above)
Comment (sglyon) [reply db/schemas/026_covid_us_mat_view.sql]: yes, thanks

File: src/cmdc_tools/datasets/nytimes/data.py (hunk 2 above)
Comment (sglyon) [reply src/cmdc_tools/datasets/nytimes/data.py:33]: ok thanks

File: db/schemas/026_covid_us_mat_view.sql (hunk 1 above)
Comment (cc7768) [reply db/schemas/026_covid_us_mat_view.sql]:
```suggestion
COMMENT ON COLUMN api.covid_us.fips IS E'The 2 digit fips code for US states and 5 digit fips code for US counties.';
```

**Pull request merged**